vcpkg_info.txt

```diff
@@ -1,2 +1,2 @@
-https://github.com/microsoft/vcpkg.git
-c168ce4e765409509c9c482faa0468c4e1ab386b
+https://github.com/ekilmer/vcpkg.git
+86b7b97100cb354ce43f4bd88a11f890929a12e3
```

Comment:
I'd prefer to use either a vcpkg fork on https://github.com/trail-of-forks or wait until I can merge the fixes in my fork to upstream.